crates/zed/src/languages/zig/highlights.scm

```diff
@@ -1,20 +1,25 @@
 [
   (container_doc_comment)
   (doc_comment)
-  (line_comment)
+
+] @comment.doc
+
+[
+    (line_comment)
 ] @comment
 
 [
   variable: (IDENTIFIER)
   variable_type_function: (IDENTIFIER)
 ] @variable
 
-parameter: (IDENTIFIER) @parameter
+;; func parametr
+parameter: (IDENTIFIER) @variable.parametr
 
 [
   field_member: (IDENTIFIER)
   field_access: (IDENTIFIER)
-] @field
+] @variable
```

Comment:
I think we usually use `@property` for struct fields / members. Would that make sense here?